content/documentation/glossary/networking/index.adoc

```diff
@@ -0,0 +1,67 @@
+---
+title: "Networking"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation/glossary"
+---
+
+= Kiali Networking Glossary
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Networking Glossary Content
+:keywords: Kiali Documentation Glossary
+:icons: font
+
+== Destination
+
+Read the <<Destination>> definition in link:https://istio.io/help/glossary/#destination[Istio Glossary]
+
+=== Destination Rule
+
+Read the <<Destination Rule>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#DestinationRule[Istio Destination Rule]
+
+
+== Endpoint
+
+A communication endpoint is a type of communication network node. It is an interface exposed by a communicating party or by a communication channel
+
+== Error Rate
+
+It's the percentage the errors in the traffic to a specific object in a <<Rate Interval>>
+
+== Gateway
+
+Read the <<Gateway>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#Gateway[Istio Gateway]
+
+== Inbound Metrics
+
+Incoming Metrics to a <<Workload>>, <<Service>> or <<Application>>
```

Comment:
Are they really incoming metrics?  I think they are metrics for incoming requests to a...

Reply:
The same, this is the nam,e opf the tab in the UI of KIALI, https://github.com/kiali/kiali-ui/blob/f3dce9570c42458d425780caa25c071d287d1122/src/pages/AppDetails/AppDetailsPage.tsx#L113

Reply:
What if you keep the title `Inbound Metrics`, but rephrase "Metrics on requests received by a given Workload, Service or Application" ?